RELEASE.md

```diff
@@ -95,7 +95,7 @@ Create a PR for the changes to be reviewed.
 
 You can use the GitHub UI to see the difference between the release branch and the latest stable release.
 
-For example: https://github.com/prometheus-operator/prometheus-operator/compare/v0.72.0...release-0.73
+For example: prometheus-operator/compare/v0.72.0...release-0.73
```

Comment:
changed here url because script was failing:
```
            RELEASE.md:98: "https://github.com/rhobs/obo-prometheus-operator/compare/v0.72.0...release-0.73" not accessible; status code 404: Not Found
                RELEASE.md:129: "https://github.com/rhobs/obo-prometheus-operator/actions/workflows/publish.yaml" not accessible; status code 404: Not Found
```

Reply:
We'll need to exclude this file from the sed command (or always revert to the upstream version).